src/plots/cartesian/layout_attributes.js

```diff
@@ -244,9 +244,9 @@ module.exports = {
         role: 'style',
         description: 'Determines whether or not the tick labels are drawn.'
     },
-    showspikes: {
+    showspike: {
```

Comment:
Alright, I can change this back. I was thinking of the plural in terms of the axes, not the data points.